### ISSUE TITLE
[Feature request] Change gnome background to album cover

### ISSUE THREAD

**Issue by conradogarciab** [JasonLG1979/gnome-shell-extensions-mediaplayer#452]
Is this possible? I wouldn't mind doing it myself but i don't have enough time to go through all the codebase myself. Some help would be awesome.

**Comment by JasonLG1979** [comment]
I'm sure one way or another it is possible (although I have never looked into it).

I see a couple problems with it though.

1. Most people do not have album art large enough to match modern screen resolutions.
2. Even if they do there's no guarantee that a player would expose such hi-res covers over the MPRIS interface.

From my experience the average size of "Big" album art is around 500 x 500 ish. And unless you got all your music from the same place or carefully curated your collection resolutions would vary.

3. Most Album art is square. Most monitors are not.

In short, it would probably look like butt.

**Comment by JasonLG1979** [comment]
If you wanted something like that it would probably be better as a separate extension.
